packages/webchat/test/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "../../../tsconfig.packages.json",
+  "compilerOptions": {
+    "noEmit": true,
+    "rootDir": "..",
+
+    // Because original webchat was written with decorators
+    "experimentalDecorators": true,
+
+    "jsx": "react",
+    "lib": ["dom", "es2021"],
+
+    // be explicit about types included
+    // to avoid clashing with Jest types
+    "types": ["cypress"]
+  },
+  "include": ["../../../node_modules/cypress", "../test/**/*", "../src"]
+}
```

Comment:
The include here normally lists `"../src/**/*"` not `"../src"`. Is there any reason for it being different?

Reply:
We don't need to include anything from the source except the typings (`"../src"`), the one from cypress is recommended by them for some reason, and lastly, we include all the test files.